src/main/java/com/ldtteam/structurize/util/TickedWorldOperation.java

```diff
@@ -242,7 +260,7 @@ public boolean apply(final ServerWorld world)
      */
     private boolean run(final ServerWorld world)
     {
-        final FakePlayer fakePlayer = new FakePlayer(world, new GameProfile(player == null ? UUID.randomUUID() : player.getUUID(), "placeStuffForMePl0x"));
```

Comment:
breaking change 🙃 :D

Reply:
:boar:

Reply:
at least ppl then know who's fake player tried to place stuff! xD